core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -449,6 +449,23 @@ public FailedAttemptsLimitExceededException(Throwable cause) {
         }
     }
 
+    /**
+     * Could not complete an action because it was cancelled by the user.
+     */
+    public static class ActionCancelledException extends AuthException {
```

Comment:
Maybe call this `UserCancelledException` instead?  That seems a little more clear to me, and also [matches iOS](https://github.com/aws-amplify/amplify-ios/blob/main/AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Models/Error/AWSCognitoAuthError.swift#L69).

Reply:
Done!